cmd/environment.go

```diff
@@ -21,17 +22,30 @@ func (h *Handler) Environment(ctx context.Context, req *entity.CommandRequest) e
 
 	var environment *entity.Environment
 	if len(req.Args) > 0 {
-		// Create new environment
-		environment, err = h.ctrl.CreateEnvironment(ctx, &entity.CreateEnvironmentRequest{
-			Name:      req.Args[0],
-			ProjectID: project.Id,
-		})
-		if err != nil {
-			return err
+		var name = req.Args[0]
+
+		// Look for existing environment with name
+		for _, projectEnvironment := range project.Environments {
+			if name == projectEnvironment.Name {
+				environment = projectEnvironment
+			}
+		}
+
+		if (environment != nil) {
+			fmt.Printf("%s Environment: %s\n", promptui.IconGood, ui.BlueText(environment.Name))
```

Comment:
nit: The promptui stuff should maybe be added to the ui package

Reply:
Will look into this in a separate PR as there seem to be other instances of it being used as well.